src/main/java/org/folio/rest/impl/TenantRefAPI.java

```diff
@@ -122,4 +149,30 @@ String servicePointUserFilter(String s) {
           return tl.perform(attributes, headers, vertxContext, superRecordsLoaded);
         });
   }
```

Comment:
Does this mean that RAML Module Builder executes both the `loadData` and `postTenant` methods?

What order are they processed in?

cc: @adamdickmeiss @julianladisch

Reply:
I thought the loadData is executed all the time, but when I started to implement tests, I found out that loadData is not invoked in following case:

* Create tenant with `fromModuleVersion`: `mod-inventory-storage-1.0.0` (this version specified in tests);
* Then attempt to call tenant init for `fromModuleVersion`: `20.1.0` - in this case the loadData is not executed (there is a "magic" check inside the postTenant that skips loadData if `files.length` == 1 - I assume this means no DB changes).

Maybe @adamdickmeiss, @julianladisch can advice on this, I'm happy to remove the postTenant overridden

Reply:
loadData should always been called when updating: https://github.com/folio-org/raml-module-builder#extending-the-tenant-api

Reply:
What value do you set for `toModuleVersion`?

Reply:
I've tried both `20.2.1`, `20.2.0` no result. 

But a think I mentioned, the migration seems started but with some loong delay, when the test is being destroyed.

Reply:
> But a think I mentioned, the migration seems started but with some loong delay, when the test is being destroyed.

How was this observed? From logs during the tests?

Reply:
> Create tenant with fromModuleVersion: mod-inventory-storage-1.0.0 (this version specified in tests);

Which tests define this, the general test setup or specific tests for this process?

Reply:
The migration runs asynchronously: https://github.com/folio-org/raml-module-builder#extending-the-tenant-api

Reply:
It works now! Thanks.

Reply:
How do you know it works now?

Reply:
Otherwise all tests in `org.folio.rest.api.ItemShelvingOrderMigrationServiceApiTest` are failed.

Reply:
I don't follow. Were those tests not present and passing in the first commit in this pull request?

Reply:
No, they were present, but migrations called from `postTenant` but now migrations called from `loadData` method and no need to override postTenant

Reply:
Ok. At some intermediary point, when you tried moving the code from `postTenant` to `loadData`, it stopped working and now it works again?

Reply:
Yes, seems the issue was that I did not set toModule, and it treated as current module version `20.2.0`